packages/types/src/resolver/receiver.ts

```diff
@@ -0,0 +1,8 @@
+import {ResolverNameServer} from './service'
+
+export type ResolverReceiver = {
```

Comment:
receiver is no longer a string and becomes a new data structure, 
`resolve` => domain/address/receiver to use/resolve aka URL
`as` => type, `domain` means supported domain
`selectedNameServer` defaults to the only response if === 1 aka NS 
`addressRecords` the list of addresses per name server aka A record